app/views/documents/edit.html.erb

```diff
@@ -20,10 +20,10 @@
       id: "title",
       html_for: "title-field"
     }
-    if I18n.exists?("document_types.#{@edition.document_type.id}.fields.title.guidance")
+    if t_doctype_key_exists?("document_types.#{@edition.document_type.id}.fields.title.guidance")
       title_contextual_guidance.merge!({
-        title: t("document_types.#{@edition.document_type.id}.fields.title.guidance.title"),
-        content: t("document_types.#{@edition.document_type.id}.fields.title.guidance.body_govspeak"),
+        title: t_doctype("document_types.#{@edition.document_type.id}.fields.title.guidance.title"),
```

Comment:
I think we should get the helper to do more work here: `t_doctype("document_types` is repetitive. Since the helper is specific to doctypes, it makes sense to move away from strict I18n keys, and at least lose the first element of the key.

Reply:
Agreed 👍